frame/support/procedural/src/pallet/parse/inherent.rs

```diff
@@ -0,0 +1,59 @@
+// This file is part of Substrate.
```

Comment:
I am a bit confused why there is no expand equivalent to inherent. Shouldn't module implement something for it?

Reply:
Ahh or maybe the implementation needs no transformation and thus it stays just as it is?

Reply:
yes, the reason I made inherent part of pallet module is to be able later to give this information to construct_runtime with something like https://github.com/paritytech/substrate/pull/6494 but automatically generated by macro.